python/paddle/v2/fluid/layers/device.py

```diff
@@ -4,19 +4,22 @@
 
 from ..layer_helper import LayerHelper
 from ..framework import unique_name
+from ..registry import autodoc
 
 __all__ = ['get_places']
 
 
-def get_places(device_count=0, device_type="CPU"):
+@autodoc
+def get_places(device_count=None, device_type=None):
     helper = LayerHelper('get_places', **locals())
     out_places = helper.create_variable(name=unique_name(helper.name + ".out"))
+    attrs = dict()
```

Comment:
Do not set `device_count` and `device_type` if the user does not specify them.

Because C++ side will set the default value. Do not set default value twice, it makes code harder to maintain.